app.js

```diff
@@ -13,6 +13,49 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function stateToUrl(state) {
+  var qsParams = Object.keys(state);
+
+  if (!qsParams.length) {
+    return "";
+  }
+
+  return "?" + qsParams.map(function(param) {
```

Comment:
We could use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams here